### ISSUE TITLE
Add files.json description root level key

### ISSUE THREAD

**Issue by cberys** [cchdo/cchdo.ucsd.edu#19]
I am proposing to add a new root level key of "description" to the files.json as well as what has been proposed for the cruise.json (see issue #14 ). This key would contain a single string with any plain text. The purpose would be to create a description for the "list" command in a uow to display.

This new key would be OPTIONAL.

Simple example:

{
  "expocode": "33RRYYYYMMDD",
  "description": "PH update submitted by Bob Key on 2014-05-06, replaces the submission from 2013-02-03",
  //... all the other needed keys for a valid files.json
}

**Comment by DocOtak** [comment]
I think this would also be a good place for including the "argo calibration suitability" comments from Jim:

Example:

``` javascript
{
  "role":"argo",
  "description": "These data have been found to be of high quality, and up to GO-SHIP standards."
// ... the other keys needed for valid files.json
}
```

**Comment by DocOtak** [comment]
We could also generate the "argo data report" from this information, all we need now is "number of profiles" to somehow be included.

**Comment by DocOtak** [comment]
Fixed with f284c2e
